music_assistant/providers/subsonic_scrobble/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "type": "plugin",
+  "domain": "subsonic_scrobble",
+  "name": "Subsonic Scrobbler",
+  "description": "Report your music playback back to Subsonic server",
+  "codeowners": ["@clusters"],
+  "documentation": "",
+  "multi_instance": false,
+  "builtin": false,
+  "requirements": [
+    "py-opensonic==5.3.1"
```

Comment:
If this goes in before https://github.com/music-assistant/server/pull/2160, I will update that pull request to pull this plugin forward as well.

Reply:
I could change it to `py-opensonic>=5.3.1,<7.0.0`, I don't see any risk in doing so.

Maybe it would be wise even to set it to `py-opensonic>=5.3.1`, as it just requires one type of that and it would minimize the risk of a dependency conflict with the music provider :)

Reply:
The scrobble method didn't change in the churn, but in my conversion to be 'more pythonic' the library is fundamentally not backwards compatible. I think the solution @marcelveldt suggested is the correct one. This plugin should depend on the Open Subsonic music provider, even better if we refuse to configure this one without having the provider configured.

Reply:
was changed to depends_on. This also fixed the issue with the race condition on startup 👍